From 5af3b99f5d6bf5b2bfda14cc63bd54d82a6cabb0 Mon Sep 17 00:00:00 2001
From: Peter Bergin <peter@berginkonsult.se>
Date: Thu, 23 Oct 2025 17:58:04 +0200
Subject: [PATCH] tegra264-ufs-provision.dts: make build reproducible

Remove information about filename and built time to make
this build in a reproducible way.

Upstream-Status: Inappropriate [OE specific]

Signed-off-by: Peter Bergin <peter@berginkonsult.se>
Signed-off-by: Peter Bergin <peter.bergin@1x.tech>
---
 nv-platform/tegra264-ufs-provision.dts | 2 --
 1 file changed, 2 deletions(-)

diff --git a/nv-platform/tegra264-ufs-provision.dts b/nv-platform/tegra264-ufs-provision.dts
index 9ec5af5..d225837 100644
--- a/nv-platform/tegra264-ufs-provision.dts
+++ b/nv-platform/tegra264-ufs-provision.dts
@@ -6,8 +6,6 @@
 / {
 	overlay-name = "Tegra264 UFS Provisioning overlay";
 	compatible = "nvidia,tegra264";
-        nvidia,dtsfilename = __FILE__;
-        nvidia,dtbbuildtime = __DATE__, __TIME__;

 	/* Enable UFS provisioning */
 	fragment@0 {
--
2.34.1

